### DISCUSSION TITLE
GCD Comictagger plugin

### DISCUSSION THREAD

**Discussion by mizaki** [GrandComicsDatabase/gcd-django#614] [General]
As mentioned in https://github.com/GrandComicsDatabase/gcd-django/discussions/608 I've have a written a  [GCD plugin](https://github.com/mizaki/comictagger-gcd-talker) for use with [Comictagger](https://github.com/comictagger/comictagger). I wanted to let you know so if you have any issues (or just wanted to check it out), you know where to look.

There is an option to download covers (which will be cached for future use), if this is a problem, I'll remove it. It is designed to download the minimum.

Anything else is also welcome.

**Comment by jochengcd** [comment]
As long as there is no bulk download of covers, anything which uses our webpage is out of our control in the end, so covers should be fine. If there is a bulk capability, the update rate would need to be limited, otherwise cloudflare will be on.

**Reply by mizaki** [reply]
I'll layout what I believe is a worst case scenario:

The minimum required for "auto-tagging" is a name and issue number. So, taking `batman` and issue `1` with the current DB I have, there are 125 results. With scraping and image download and having bad luck, that one search could be 250 requests. 

And, of course, people could want to tag a large number comics.

Given that, what would be the rate limit per request? We already honour API rate limits so happy to put one in.

**Comment by Charlotte-br560** [comment]
Great work on the GCD plugin for Comictagger! Looking forward to trying it out. The option to download covers sounds handy, especially with the caching feature. I don't see any issue with it, but thanks for offering to remove it if needed. I'll definitely check it out and let you know if I encounter any issues.
